exporter/dynatraceexporter/README.md

```diff
@@ -1,5 +1,11 @@
 # Dynatrace Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```

Comment:
We are also in the aws distro. Should that be listed here?

Reply:
I think that makes sense.

Reply:
@codeboten how do you feel about linking to non-opentelemetry distributions?

Reply:
@Frapschen please add the AWS distribution